app/src/release/res/values/strings.xml

```diff
@@ -10,7 +10,7 @@
     <string name="OCTOPUS_GRAPHQL_URL" translatable="false">https://apollo-router.prod.hedvigit.com</string>
     <string name="BASE_URL" translatable="false">https://giraffe.hedvig.com</string>
     <string name="WEB_BASE_URL" translatable="false">https://www.hedvig.com</string>
-    <string name="HANALYTICS_URL" translatable="false">https://hanalytics-production.herokuapp.com</string>
+    <string name="HANALYTICS_URL" translatable="false">https://hanalytics.prod.hedvigit.com/</string>
```

Comment:
Are you sure you want the trailing slash here? :)

Reply:
I am definitely sure that I do *not* want it 😅
Good eye, it was a mistake from copy-pasting, thanks for noticing!